docs/pages/access-controls/guides/moderated-sessions.mdx

```diff
@@ -182,139 +236,85 @@ Teleport > Connecting to prod.teleport.example.com over SSH
-- `k8s` policies apply to all Kubernetes sessions on clusters connected to Teleport.
-
 ### Participant modes
 
 A participant joining a session will always have one of three modes:
 
-- `observer`: Allows read-only access to the session. You can view output but cannot control the session in any way nor send any input.
-- `moderator`: Allows you to watch the session. You can view output and forcefully terminate or pause the session at any time, but can't send input.
+- `observer`: Allows read-only access to the session. You can view output but 
+  cannot control the session in any way nor send any input.
+- `moderator`: Allows you to watch the session. You can view output and forcefully 
+  terminate or pause the session at any time, but can't send input.
 - `peer`: Allows you to collaborate in the session. You can view output and send input.
 
-When joining a session with `tsh join` or `tsh kube join`, a user can specify a
-participant mode with the `--mode <mode>` flag , where the mode is one of `peer`,
-`moderator` or `observer`. By default, the mode is `observer`.
-
-A participant may leave a session with the shortcut `^c` (Control + c) while in observer or
-moderator mode. When in moderator mode, a participant may also forcefully
-terminate the session at any point in time with the shortcut `t`.
-
-### Require policy count
+If you join a session with `tsh join` or `tsh kube join`, you can specify a
+participant mode with the `--mode <mode>` command-line option, where `<mode>` is `peer`,
+`moderator`, or `observer`. The default participant mode is `observer`.
 
-Require policies can have a variable amount of users that need to match the
-filter expression in order to satisfy the policy. The `count` field of a require
-policy is a positive integer value that specifies the minimum amount of users
-this policy requires.
+You can leave a session with the shortcut `^c` (Control + c) while in observer or
+moderator mode. In moderator mode, you can also forcefully terminate the session 
+at any point in time with the shortcut `t`.
 
-### On leave
+### Multifactor authentication
 
-The `on_leave` string option in require policies is used to define what happens when a moderator leaves a session, causing a policy to no longer be satisfied.
+If `per_session_mfa` is set to `true` in role or cluster settings, Teleport requires 
+multifactor authentication checks when starting new sessions. This requirement is
+also enforced for session moderators. Therefore, moderators who want to join a session
+must have configured a device for multifactor authentication.
 
-There are two possible actions to take in this scenario:
-- Terminate the session and disconnect all participants, corresponding to the `"terminate"` value.
-- Pause the session and stop any input/output streaming until the policy is satisfied again, corresponding to the `"pause"` value.
+Every 30 seconds, Teleport moderating users who have joined the terminal session to 
```

Comment:
> Every 30 seconds, Teleport moderating users who have joined the terminal session to  authenticate in the next 15 seconds

I'm not sure what this is supposed to mean—looks like there is a missing or misused word here.